get-poetry.py

```diff
@@ -226,10 +226,6 @@ def expanduser(path):
 {poetry_home_bin}
 
 {platform_msg}
-
-You can uninstall at any time with `poetry self uninstall`,
-or by executing this script with the --uninstall option,
```

Comment:
The `--uninstall` option of the `get-poetry.py` script is implemented so we can leave this line

Reply:
You mean `poetry self uninstall` isn't implemented but `get-poetry.py --uninstall` is?